z/zlib/zlib-1.2.11-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'zlib'
+version = '1.2.11'
```

Comment:
There is already a zlib 1.2.12 for GCCcore 11.3.0 in the upstream repo; why install an older version? There is e security bug introduced in zlib 1.2.12 that got fixed in 1.2.13 (latest), so if you do want to include something new I would suggest trying 1.2.13....